corehq/motech/dhis2/tests/test_entities_helpers.py

```diff
@@ -43,6 +59,114 @@ def test_bad_data_type(self):
             validate_tracked_entity(tracked_entity)
 
 
+class TestDhis2EntitiesHelpers(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain = create_domain(DOMAIN)
+        location_type = LocationType.objects.create(
+            domain=DOMAIN,
+            name='test_location_type',
+        )
+        cls.location = SQLLocation.objects.create(
+            domain=DOMAIN,
+            name='test location',
+            location_id='test_location',
+            location_type=location_type,
+            latitude='-33.6543',
+            longitude='19.1234',
+            metadata={LOCATION_DHIS_ID: "dhis2_location_id"},
+        )
+        cls.user = WebUser.create(DOMAIN, 'test', 'passwordtest', None, None)
+        cls.user.set_location(DOMAIN, cls.location)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete(deleted_by=None)
```

Comment:
This is missing the `deleted_by_domain` argument (https://github.com/dimagi/commcare-hq/blob/cc0484b7eb84ece50b8a323c4be1e0e95bb8eccf/corehq/apps/users/models.py#L1098). I'm not quite sure how the build passed.

Reply:
https://github.com/dimagi/commcare-hq/pull/30314

Reply:
Woah, this is strange. This had to pass local testing as well as Travis...interesting...